build-docker.sh

```diff
@@ -1,27 +1,36 @@
 #!/usr/bin/env bash
-set -e
+set -e -o pipefail
 
-CONTAINER_NAME=trezor-firmware-env.nix
+cd "$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
@prusnak Have you noticed this? I wanted guaranteed starting working dir. It is another bash-ism :-) Reason: `${BASH_SOURCE[0]}` is better than using `$0` for some obscure reasons (e.g. someone could symlink the script and try to run it via a symlink).

Reply:
Yes, noticed. Let's keep.